lib/DOMMatrix.js

```diff
@@ -63,68 +61,441 @@ function parseTransform(tform) {
+            m[12] * m[3] * m[9]
+
+    inv[13] = m[0]  * m[9] * m[14] -
+            m[0]  * m[10] * m[13] -
+            m[8]  * m[1] * m[14] +
+            m[8]  * m[2] * m[13] +
+            m[12] * m[1] * m[10] -
+            m[12] * m[2] * m[9]
+
+    inv[2] = m[1]  * m[6] * m[15] -
+            m[1]  * m[7] * m[14] -
+            m[5]  * m[2] * m[15] +
+            m[5]  * m[3] * m[14] +
+            m[13] * m[2] * m[7] -
+            m[13] * m[3] * m[6]
+
+    inv[6] = -m[0]  * m[6] * m[15] +
+            m[0]  * m[7] * m[14] +
+            m[4]  * m[2] * m[15] -
+            m[4]  * m[3] * m[14] -
+            m[12] * m[2] * m[7] +
+            m[12] * m[3] * m[6]
+
+    inv[10] = m[0]  * m[5] * m[15] -
+            m[0]  * m[7] * m[13] -
+            m[4]  * m[1] * m[15] +
+            m[4]  * m[3] * m[13] +
+            m[12] * m[1] * m[7] -
+            m[12] * m[3] * m[5]
+
+    inv[14] = -m[0]  * m[5] * m[14] +
+            m[0]  * m[6] * m[13] +
+            m[4]  * m[1] * m[14] -
+            m[4]  * m[2] * m[13] -
+            m[12] * m[1] * m[6] +
+            m[12] * m[2] * m[5]
+
+    inv[3] = -m[1] * m[6] * m[11] +
+            m[1] * m[7] * m[10] +
+            m[5] * m[2] * m[11] -
+            m[5] * m[3] * m[10] -
+            m[9] * m[2] * m[7] +
+            m[9] * m[3] * m[6]
+
+    inv[7] = m[0] * m[6] * m[11] -
+            m[0] * m[7] * m[10] -
+            m[4] * m[2] * m[11] +
+            m[4] * m[3] * m[10] +
+            m[8] * m[2] * m[7] -
+            m[8] * m[3] * m[6]
+
+    inv[11] = -m[0] * m[5] * m[11] +
+            m[0] * m[7] * m[9] +
+            m[4] * m[1] * m[11] -
+            m[4] * m[3] * m[9] -
+            m[8] * m[1] * m[7] +
+            m[8] * m[3] * m[5]
+
+    inv[15] = m[0] * m[5] * m[10] -
+            m[0] * m[6] * m[9] -
+            m[4] * m[1] * m[10] +
+            m[4] * m[2] * m[9] +
+            m[8] * m[1] * m[6] -
+            m[8] * m[2] * m[5]
+
+    inv.forEach((v,i) => inv[i] = v/det)
+    this._values = inv
+    return this
+  }
+
+  setMatrixValue(transformList) {
+    const temp = new DOMMatrix(transformList);
+    this._values = temp._values
+    this._is2D = temp._is2D
+    return this
+  }
+
+  transformPoint(point) {
+    point = new DOMPoint(point)
+    const x = point.x;
+    const y = point.y;
+    const z = point.z;
+    const w = point.w;
+    const values = this._values;
+    const nx = values[M11] * x + values[M21] * y + values[M31] * z + values[M41] * w;
+    const ny = values[M12] * x + values[M22] * y + values[M32] * z + values[M42] * w;
+    const nz = values[M13] * x + values[M23] * y + values[M33] * z + values[M43] * w;
+    const nw = values[M14] * x + values[M24] * y + values[M34] * z + values[M44] * w;
+    return new DOMPoint(nx, ny, nz, nw)
+  }
+
+  toFloat32Array() {
+    return Float32Array.from(this._values)
+  }
+
+  toFloat64Array() {
+    return this._values.slice(0)
   }
 }
 
-DOMMatrix.fromMatrix = function (init) {
```

Comment:
Could you make these static members maybe? `static fromMatrix(init) { ... }`

Reply:
will do